src/components/build/create_build_config/Validations.js

```diff
@@ -60,9 +66,12 @@ export function checkMandatoryFields(state) {
 }
 
 export function configValidation(key, value) {
+  let response = {};
   switch (key) {
     case KEY_CR_NAME:
-      return validateNotEmpty(value);
+      response = validateNotEmpty(value);
```

Comment:
Since this is being overwritten immediately anyway it can be removed. `validateNameString` also fails if the name is empty so it catches both.